apps/design-system/content/docs/components/table.mdx

```diff
@@ -1,75 +1,178 @@
 ---
 title: Table
-description: A responsive table component for basic data.
+description: A responsive table component for presenting data inline.
 component: true
+source:
+  shadcn: true
 ---
 
-Table presents basic tabular data presentation within a [Card](/docs/components/card).
+Table is an opinionated `<table>` element for basic tabular data presentation. It is usually presented within a [Card](../components/card).
 
 <ComponentPreview name="table-demo" peekCode wide />
 
-Columns will naturally take the width of their content. Width specification is therefore not required unless you require columns to span a specific width. An example use-case may be when placing multiple Table instances on the one page, where columns on each table should optically align.
+Table is comprised of the following [Shadcn primitives](https://ui.shadcn.com/docs/components/table) which map directly to HTML counterparts:
 
-`TableHead` includes a `whitespace-nowrap` utility class to prevent its text from wrapping. `TableCell` must have any custom width specified, as there are legitimate use cases for both wrapping (or truncated) text.
+| Component      | HTML element |
+| -------------- | ------------ |
+| `Table`        | `<table>`    |
+| `TableBody`    | `<tbody>`    |
+| `TableCaption` | `<caption>`  |
+| `TableCell`    | `<td>`       |
+| `TableFooter`  | `<tfoot>`    |
+| `TableHead`    | `<th>`       |
+| `TableHeader`  | `<thead>`    |
+| `TableRow`     | `<tr>`       |
+
+These components follow standard HTML semantics and best practices. For example, `TableCaption` (just like `<caption>`) should only ever contain inline elements as children, and be placed at the start of a `<table>` irrespective of visual positioning.
```

Comment:
I found this quite helpful to know when doing https://github.com/supabase/supabase/pull/41217, so figured I’d add it to the docs.